src/software/backend/simulation/simulator_robot.h

```diff
@@ -218,6 +218,22 @@ class SimulatorRobotSingleton
     static float getMotorSpeedBackRight();
     static float getMotorSpeedFrontRight();
 
+    /**
+     * Sets the motor to coast (spin freely)
+     */
+    static void coastMotorBackLeft();
+    static void coastMotorBackRight();
+    static void coastMotorFrontLeft();
+    static void coastMotorFrontRight();
+
+    /**
+     * Sets the motor to brake (act against the current direction of rotation)
```

Comment:
We might need to elaborate on what 'braking' means.

It is trying to achieve zero velocity?

Reply:
TBD? I don't know what the expected behaviour from the robot side is and I'm not confident enough to even take a guess. I suspect it's just not free-wheeling, so the back-emf is all that's acting against it.

Reply:
I'm just wondering if it is breaking, buy how much?

Reply:
That'll depend on the motor constants (speed and torque constants). I'm not sure if we want to strictly define that here or not, just because it will be hardware dependent. (This is all assuming my guess for what this does is correct)

Reply:
Yeah, our maximums are based on the motor parameters. I mean is it trying to brake at 100% effort, or 20% ect

Reply:
But out of scope